src/Components/PaperEditor/TranscriptsContainer/LabelsList/index.js

```diff
@@ -16,24 +16,27 @@ import {
 
 import LabelModal from './LabelModal.js';
 import { randomColor } from './css-color-names.js';
+import PropTypes from 'prop-types';
 
-class LabelsList extends Component {
-  constructor(props) {
-    super(props);
+const LabelsList = (props) => {
+  const labelsOptions = props.labelsOptions;
+  const isLabelsListOpen = props.isLabelsListOpen;
+  const onLabelDelete = props.onLabelDelete;
+  const onLabelCreate = props.onLabelCreate;
+  const onLabelUpdate = props.onLabelUpdate;
 
-    this.state = {
-      // isLabelsListOpen: false,
-      isLabelmodalShown: false
-    };
-  }
+  const [ isLabelmodalShown, setIsLabelmodalShown ] = useState(false);
```

Comment:
Being nit-picky: keeping naming consistent would be good
```suggestion
  const [ isLabelModalShown, setIsLabelModalShown ] = useState(false);
```